scene/gui/flow_container.cpp

```diff
@@ -203,7 +204,21 @@ void FlowContainer::_resort() {
 			}
 		}
 
-		if (vertical) { /* VERTICAL */
+		bool is_unsupported_texture_rect = false;
+		TextureRect *trect = Object::cast_to<TextureRect>(child);
+		if (trect) {
+			TextureRect::ExpandMode mode = trect->get_expand_mode();
+			if (mode == TextureRect::EXPAND_FIT_WIDTH || mode == TextureRect::EXPAND_FIT_WIDTH_PROPORTIONAL ||
+					mode == TextureRect::EXPAND_FIT_HEIGHT || mode == TextureRect::EXPAND_FIT_HEIGHT_PROPORTIONAL) {
+				is_unsupported_texture_rect = true;
+			}
+		}
+
+		if (is_unsupported_texture_rect && lines_data.size() > 1) {
```

Comment:
`lines_data.size() > 1` can be done before checking TextureRect; there is no reason to check the class and expand mode if it's known to be irrelevant.

Reply:
Updated to do size check first